infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -274,23 +275,24 @@ private void initializationTask() {
             }
 
             // Initiating periodic task to poll for failures.
-            try {
```

Comment:
Please combine the logic of sequencer bootstrap for the case where it is a fresh start or recovery.